setup.py

```diff
@@ -118,7 +119,9 @@ def run(self):
                         'python-dateutil==2.6.1',
                         'pympler==0.5',
                         'packaging==19.0',
+                        'ursacl==0.1.0',
```

Comment:
Why is it called `ursacl`? Should it be `python-ursa`?

Reply:
+1